Lib/pdb.py

```diff
@@ -1614,6 +1646,12 @@ def set_trace(*, header=None):
 # Post-Mortem interface
 
 def post_mortem(t=None):
```

Comment:
Here's another discrepancy between the docs and the function signature. This one is a bit less significant, but I find it odd that the actual function uses `t` as the function name while the documentation uses *traceback*.

I'd be in favor of adjusting the documentation here, as it could lead to issues if a user tries to pass a keyword argument to *traceback*. For example: `pdb.post_mortem(traceback=tb)` would lead to a rather confusing `TypeError`.

Once again, not directly related to the PR but worth noting.